doc/nrf/ug_nrf70_features.rst

```diff
@@ -0,0 +1,55 @@
+.. _ug_nrf70_features:
+
+Features of nRF70 Series
+########################
+
+.. contents::
+    :local:
+    :depth: 2
+
+Introduction
+************
+
+The nRF70 Series of wireless companion ICs add Wi-Fi capabilities to another System-on-Chip (SoC), Microprocessor Unit (MPU), or Microcontroller (MCU) host device.
+It implements the Physical (PHY) and Medium Access Controller (MAC) layers of the 802.11 protocol stack, while the higher layers of the networking stack run on the host device.
```

Comment:
802.11 => IEEE802.11

Reply:
As this PR is merged, these two suggestions will be fixed in https://github.com/nrfconnect/sdk-nrf/pull/9905/files.

Reply:
Fixed in https://github.com/nrfconnect/sdk-nrf/pull/9905